pkg/burner/waiters.go

```diff
@@ -247,71 +267,84 @@ func waitForBuild(ns string, maxWaitTimeout time.Duration, expected int, limiter
 	})
 }
 
-func waitForJob(ns string, maxWaitTimeout time.Duration, limiter *rate.Limiter) {
-	gvr := schema.GroupVersionResource{
+func waitForJob(ns string, maxWaitTimeout time.Duration, obj object, limiter *rate.Limiter) {
+	obj.gvr = schema.GroupVersionResource{
 		Group:    "batch",
 		Version:  "v1",
 		Resource: "jobs",
 	}
-	verifyCondition(gvr, ns, "Complete", maxWaitTimeout, limiter)
+	if obj.WaitOptions.ForCondition != "" {
+		obj.WaitOptions.ForCondition = "Complete"
+	}
+	verifyCondition(ns, maxWaitTimeout, obj, limiter)
 }
 
-func verifyCondition(gvr schema.GroupVersionResource, ns, condition string, maxWaitTimeout time.Duration, limiter *rate.Limiter) {
+func verifyCondition(ns string, maxWaitTimeout time.Duration, obj object, limiter *rate.Limiter) {
 	var uObj types.UnstructuredContent
 	wait.PollUntilContextTimeout(context.TODO(), 10*time.Second, maxWaitTimeout, true, func(ctx context.Context) (done bool, err error) {
```

Comment:
Lets drop the interval period to 1s as well to be consistent with the other wait functions.
 
```suggestion
	wait.PollUntilContextTimeout(context.TODO(), time.Second, maxWaitTimeout, true, func(ctx context.Context) (done bool, err error) {
```

Reply:
Changed as suggested.